### ISSUE TITLE
Linux Rails page shows content about Atom

### ISSUE THREAD

**Issue by suzannehamilton** [makersacademy/preparetocode#61]
The [Useful Linux](http://www.preparetocode.io/linux/useful/) page links to the [Linux Rails](http://www.preparetocode.io/linux/useful/rails.html) page, but it shows information about Atom rather than Rails.

Fixing this might involve actually writing the documentation for Rails. :)

**Comment by leggsimon** [comment]
Yep, it links to the correct [html file](https://github.com/makersacademy/preparetocode/blob/master/source/linux/useful/rails.html.erb) but the content there is just completely wrong. This is really not good. 

There are 3 options as I see it;
- figure out how to remove the intricately intertwined link at http://www.preparetocode.io/linux/useful/ 
- Update the actual page to be about rails
- Show a 404/ Under construction page.

My preference would be #3 for a quick fix but we do need those installation steps eventually

**Comment by leggsimon** [comment]
Would you like to give it a go @suzannehamilton ?

**Comment by suzannehamilton** [comment]
I'm afraid I don't have a Linux machine (or very much time at the moment!) I just spotted it when I was looking for beginner issues for the Codebar 24 Pull Requests event yesterday.

**Comment by nikeshashar** [comment]
@leggsimon If #66 is happening then we can worry about this later?
